Content.Server/Chat/SuicideSystem.cs

```diff
@@ -123,19 +135,52 @@ private bool EnvironmentSuicideHandler(EntityUid victim, SuicideEvent suicideEve
             return false;
         }
 
-        private void ApplyDeath(EntityUid target, SuicideKind kind)
+        private void ApplyLethalDamage(EntityUid target, DamageSpecifier? damage)
         {
-            if (kind == SuicideKind.Special)
+            if (!TryComp<DamageableComponent>(target, out var damagable) || !TryComp<MobThresholdsComponent>(target, out var thresholds))
                 return;
 
-            if (!_prototypeManager.TryIndex<DamageTypePrototype>(kind.ToString(), out var damagePrototype))
+            var lethalAmountOfDamage = thresholds.Thresholds.Keys.Last() - damagable.TotalDamage;
+
+            if (damage == null)
+            {
+                var damagePrototype = _prototypeManager.Index<DamageTypePrototype>("Blunt");
+                damage = new DamageSpecifier(damagePrototype, lethalAmountOfDamage);
+            }
+
+            var finalDamage = new DamageSpecifier(damage);
+            finalDamage.DamageDict.Remove("Structural");
```

Comment:
Add a comment about why you're removing structural damage and why you need the total; (to split between each valid damage type)